### PR TITLE
Minkee contactscreen

### PR DESCRIPTION
**Pull request by minkeesung**
@CFDevelopment @masterrom @GregTheGreek

### CODE REVIEW

File: src/actions/ActionCreator.js [hunk 1]
```diff
@@ -148,4 +158,3 @@ export function enterDebug() {
     dispatch({ type: actionTypes.DEBUG_MODE, payload: '' });
```
Comment (priom) [inline src/actions/ActionCreator.js]: @minkeesung just a note, we shouldn't be running debug mode in production. We r going to performance issue on the user end.

File: src/screens/main/menu/contacts/ContactsTab.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+import SelectedContact from './SelectedContact'
+
+
+/**
+ * Screen that displays all the contacts that have been added to
+ * the wallet
+ */
+class ContactsTab extends Component {
+
+  /**
```
Comment (priom) [inline src/screens/main/menu/contacts/ContactsTab.js]: If the commented code is not going to be used in future then please clean it up.

**Comment by CFDevelopment** [comment]
we need redux persist to be integrated

**Comment by priom** [comment]
I meant u can integrate it without using DEBUG_MODE

**Comment by GregTheGreek** [comment]
@priom

**Comment by priom** [comment]
@GregTheGreek

**Pull request merged**